lib/request.js

```diff
@@ -165,6 +180,24 @@ Object.defineProperties(Request.prototype, {
       return this[kRouteContext].config.url
     }
   },
+  routeOptions: {
+    get () {
+      const context = this[kRouteContext]
+      const routeLimit = context._parserOptions.limit
+      const serverLimit = context.server.initialConfig.bodyLimit
+      const options = {
+        method: context.config.method,
+        url: context.config.url,
+        bodyLimit: (routeLimit || serverLimit),
+        attachValidation: context.attachValidation,
+        logLevel: context.logLevel
+      }
+      return readOnly(options)
+    },
+    set () {
+      throw new Error('routerOptions is immutable')
+    }
```

Comment:
Not needed. When it doesn't have a `setter`, JavaScript will throw properly.

```suggestion
```

Reply:
done 👍